dashboard/src/app/[locale]/public/[dashboardId]/campaign/page.tsx

```diff
@@ -0,0 +1 @@
+export { default } from '@/app/(protected)/dashboard/[dashboardId]/campaign/page';
```

Comment:
When re-exporting the pages like this the links in our dashboard will break, which is currently solved by sending the base path to the sidebar, etc., but there may also be some "deeper" issues where we (in the future) may forget to add a page, or so.
I just investigated a little and found this NextJS feature ["rewrites"](https://nextjs.org/docs/app/api-reference/config/next-config-js/rewrites), which seems to be able to potentially solve this, without having to actually re-export the pages, by just routing `/[locale]/public/[dashboardId]` --> `/dashboard/[dashboardId]`
There may of course be some drawbacks from doing this, however, it might be worth looking into if you haven't already 😄